sources/web/datalab/static/datalab.js

```diff
@@ -714,6 +714,36 @@ function initializeNotebookApplication(ipy, notebook, events, dialog, utils) {
 }
 
 
+function initializeEditApplication(ipy, editor) {
+  function navigateAlternate(alt) {
+    var url = document.location.href.replace('/edit', alt);
+    url = url + '?download=true';
+
+    if (!editor.clean) {
+      var w = window.open('');
```

Comment:
Yes, that's (presumably) what it was for.

I was following the pattern in the `initializeNotebookApplication` method's `navigateAlternate`, but screwed that up and failed to use the `w` variable.

Not using the blank window does not seem to have impaired usability, so I'm just going to remove it entirely.